### ISSUE TITLE
Make Bolt 4 compatible with Composer 2

### ISSUE THREAD

**Issue by bobdenotter** [bolt/core#1492]
At first glance it seems that `Drupol/composer-packages` and `vaimo/binary-chromedriver` need an update. See:

 - [ ] https://github.com/drupol/composer-packages/issues/10
 - [ ] https://github.com/vaimo/binary-chromedriver/pull/3
 - [x] Easy Coding Standard needs to be updated: #1488

**Comment by bobdenotter** [comment]
This is becoming higher priority, since Composer is now showing this notice: 

```
Warning from https://repo.packagist.org: You are using an outdated version of Composer. Composer 2.0 is about to be released and the older 1.x releases will self-update directly to it once it is released. To avoid surprises update now to the latest 1.x version which will prompt you before self-updating to 2.x.
```

**Comment by bobdenotter** [comment]
Closed by #2049
